app/controllers/admin/content/system-images/subtopics.js

```diff
@@ -0,0 +1,20 @@
+import Ember from 'ember';
+
+const { Controller, computed } = Ember;
+export default Controller.extend({
+  subTopics: computed('model.currentTopicID', function() {
```

Comment:
This filtering is not needed anymore. As once we implement api, the model itself will give only the subtopics for that selected topic.